app/code/Magento/Config/Model/Config.php

```diff
@@ -237,13 +237,14 @@ private function getField(string $sectionId, string $groupId, string $fieldId):
      * Get field path
      *
      * @param Field $field
+     * @param string $fieldId Need for support of clone_field feature
      * @param array &$oldConfig Need for compatibility with _processGroup()
      * @param array &$extraOldGroups Need for compatibility with _processGroup()
      * @return string
      */
-    private function getFieldPath(Field $field, array &$oldConfig, array &$extraOldGroups): string
+    private function getFieldPath(Field $field, string $fieldId, array &$oldConfig, array &$extraOldGroups): string
```

Comment:
I'm not sure that changing parameters order and adding a new parameter in the middle of the sequence a good idea. Current implementation will broke all usages of `getFieldPath ` method because array will be passed as second parameter instead of string.

Reply:
You're right, I agree with you. But in this case I think i can change the order of the parameters as I did because it is a **private** method. If it were anything else (protected or public) changing parameters order is of course, unacceptable. There are only 2 implementations of the `getFieldPath` method in `Magento/Config/Model/Config` which I replaced in the commit.